Memoria.Patcher/Program.cs

```diff
@@ -43,6 +43,9 @@ static void Main(String[] args)
                             process?.WaitForExit();
                         }
 
+                        // Send a command "FF9_Launcher.exe" even when the Steam overlay fix redirects to "FF9_Launcher.fix"
+                        if (launcherProcessPath.EndsWith(".fix"))
+                            launcherProcessPath = launcherProcessPath.Substring(0, launcherProcessPath.Length - 4) + ".exe";
```

Comment:
Couldn't you use `launcherProcessPath = Path.ChangeExtension(launcherProcessPath, ".exe")`? It does the same thing but it's a bit more readable.